frontend/src/scenes/billing/billing-utils.spec.ts

```diff
@@ -118,7 +119,9 @@ describe('convertAmountToUsage', () => {
             return
         }
         if (billingJson.products[0].tiers) {
-            expect(convertAmountToUsage(mapping.amount, [billingJson.products[0].tiers])).toEqual(mapping.usage)
+            expect(
+                convertAmountToUsage(mapping.amount, [billingJson.products[0].tiers], mapping.discountPercent)
+            ).toEqual(mapping.usage)
```

Comment:
oops I don't think this unit test should still remain, u8sage in this case would be infinite and that's not something we handle / it should never get here.

Reply:
Good point, thanks.